package.json

```diff
@@ -16,8 +16,6 @@
     "@supabase/supabase-js": "^2.45.3",
     "next": "14.2.7",
     "nodemailer": "^6.9.15",
-    "react": "^18",
```

Comment:
i think you still need react and react dom. Was just suggesting to remove resend. new to these comments my bad if it wasn't clear
@SnugTop